lib/pure/net.nim

```diff
@@ -453,7 +454,7 @@ when defineSsl:
     GC_ref(data)
 
   # http://simplestcodings.blogspot.co.uk/2010/08/secure-server-client-using-openssl-in-c.html
-  proc loadCertificates(ctx: SSL_CTX, certFile, keyFile: string) =
+  proc loadCertificates*(ctx: SSL_CTX, certFile, keyFile: string) =
```

Comment:
Is there a reason this is now exported? I would prefer to keep procs that work on SSL_CTX hidden.

Reply:
Good point, I'll hide it.